test/SourceKit/CursorInfo/cursor_symlink.swift

```diff
@@ -4,5 +4,8 @@
 // RUN: %sourcekitd-test -req=cursor -pos=1:5 %t.dir/linked.swift -- %t.dir/real.swift | %FileCheck %s
 // RUN: %sourcekitd-test -req=cursor -pos=1:5 %t.dir/real.swift -- %t.dir/linked.swift | %FileCheck %s
 
+// We can't resolve symlinks on a substituted drive without risking MAX_PATH issues
+// REQUIRES: !windows_substituted_drive
```

Comment:
I'm still confused as to why these are needed. Isn't the swap to `resolveSymbolicLinks` to fix this?

Reply:
If we are working under `S:`, a substituted drive pointing to `C:\Users\swift-ci\jenkins\workspace\win64-vs2022` (a real scenario in CI), then we won't be able to resolve `S:\SymbolicLinkSource` to `S:\SymbolicLinkTarget` because doing so would require calling realpath, which would actually resolve to `C:\Users\swift-ci\jenkins\workspace\win64-vs2022\SymbolicLinkTarget`, and we would risk running into `MAX_PATH` issues that the `S:` substituted drive is trying to avoid.